torch/onnx/_internal/fx/op_validation.py

```diff
@@ -231,7 +231,7 @@ def _fx_args_to_torch_args(
                 )
         elif isinstance(arg, Sequence):
             wrapped_args.append(_fx_args_to_torch_args(arg))
-        elif isinstance(arg, (int, float, torch.dtype)):
```

Comment:
Could you explain why this change?

Reply:
In aten::convolution, the bias can be passed as None in args...

Reply:
The whole story is that resnet18 raises another issue which only fails when op_level_debug is True. And the reason is simply that the None arg is not in any condition, and go straight to the ValueError. So now, we just append it to the args as if it's normal value.